clang/lib/Gold/GoldLexer.cpp

```diff
@@ -341,7 +369,9 @@ Token CharacterScanner::matchNewline() {
 }
 
 Token CharacterScanner::matchLineComment() {
-  assert(nextCharacterIs('#'));
+  // assert(nextCharacterIs('#'));
```

Comment:
can you clean up some of the comments in here? it's already a big change to look through

Reply:
Fixed, but not pushed yet.